apps/api/scripts/backfill-internal-tx-claims.ts

```diff
@@ -230,15 +238,22 @@ async function findMissingClaims() {
     return;
   }
 
-  // Get all accounts from the database
-  const dbAccounts = await db
-    .select({ account: convictionClaims.account })
+  // Get all account+season pairs from the database
+  const dbClaims = await db
+    .select({
+      account: convictionClaims.account,
+      season: convictionClaims.season,
+    })
     .from(convictionClaims)
     .execute();
 
-  const accountSet = new Set(dbAccounts.map((a) => a.account.toLowerCase()));
+  const accountSeasonSet = new Set(
+    dbClaims.map((c) => `${c.account.toLowerCase()}:${c.season}`),
```

Comment:
needed to update this to consider seasons when checking if a wallet address has claimed